plugins/common.go

```diff
@@ -0,0 +1,12 @@
+package plugins
+
+import (
+	"github.com/gluster/glusterd2/servers/rest/route"
+	"github.com/prashanthpai/sunrpc"
+)
+
+type Gd2Plugin interface {
+	SunRpcProgram() sunrpc.Program
```

Comment:
This will mean one plugin can have only one RPC program, which is completely okay for our uses I guess 👍